### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by vermasavita**
What are the changes?

- Add auth for login
- Add sign up auth
- Add routehandler
- Add sidebar to all the pages
- Fix minnor style

### CODE REVIEW

File: src/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -18,11 +30,17 @@ const Navbar = () => {
       </div>
 
       <div className="action-icon">
-        <button className="btn">
-          <Link className="link" to="/login">
-            Login
-          </Link>
-        </button>
+        {localStorage.getItem("token") ? (
```
Comment (sanjitsarkar) [inline src/components/navbar/Navbar.jsx:33]: Keep the token in a state

File: src/pages/Liked/LikedVideos.jsx [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
-            <LikedVideoCard/>
+          <LikedVideoCard />
+          <LikedVideoCard />
+          <LikedVideoCard />
+          <LikedVideoCard />
+          <LikedVideoCard />
+          <LikedVideoCard />
+          <LikedVideoCard />
+          <LikedVideoCard />
+          <LikedVideoCard />
```
Comment (sanjitsarkar) [inline src/pages/Liked/LikedVideos.jsx:21]: You can use map here

**Review by sanjitsarkar — APPROVED** [review]
Code looks clean and easy to understand

File: src/pages/Liked/LikedVideos.jsx (hunk 2 above)
Comment (vermasavita) [reply src/pages/Liked/LikedVideos.jsx:21]: Yes, will work on rest functionality

**Review by starver20 — APPROVED** [review]
Auth is working as intended. Good to merge.

**Review by Jay-Gadhiya — APPROVED** [review]

**Pull request merged**